logging_demo/test/logging_demo_main_debug_severity.regex

```diff
@@ -0,0 +1 @@
+\[DEBUG\] \[logger_usage_demo\]: Count divides into 12 \(function evaluated to true\)
```

Comment:
Doest this need to be a regex? it seems like it could be a string to match like the file below

Reply:
if it's a txt it can't have any messages printed before this one gets matched or it fails